modules/pipeline/spec/pipeline_extra.go

```diff
@@ -115,8 +115,9 @@ type PipelineExtraInfo struct {
 }
 
 type QueueInfo struct {
-	QueueID        uint64 `json:"queueID"`
-	CustomPriority int64  `json:"customPriority"`
+	QueueID               uint64  `json:"queueID"`
+	CustomPriority        int64   `json:"customPriority"`
+	PriorityChangeHistory []int64 `json:"priorityChangeHistory,omitempty"`
```

Comment:
add a comment or adjust field name to clarify original first or latest first.

Reply:
done